float32 position
bool is_counterclockwise
---
bool success